khmer/khmer_args.py

```diff
@@ -360,12 +361,15 @@ def build_graph_args(descr=None, epilog=None, parser=None):
     group = parser.add_mutually_exclusive_group()
     group.add_argument('--max-tablesize', '-x', type=float,
                        default=DEFAULT_MAX_TABLESIZE,
-                       help='upper bound on tablesize to use; overrides ' +
+                       help='upper bound on tablesize to use; overrides '
                        '--max-memory-usage/-M')
```

Comment:
One thing you can do is add `(` `)` around the multi line string and indent the second line to line up with the one above. I find it helps with making it clear that these strings were meant to be joined together. Not sure what the style gods say regarding implicit/explicit joining of strings across lines.